### DISCUSSION TITLE
Introduce a third panel for tours

### DISCUSSION THREAD

**Discussion by alexsegura** [coopcycle/coopcycle-web#3636] [General]
See #3552

This would solve most of the problems of tour edition (reordering, drag'n'dropping a new task), also in terms of implementation with `react-beautiful-dnd` draggable/droppable structure. 

---

In order to solve the problem of _Tour_ edition we are ading a third panel to the dashboard. This will be the **Unassigned Tours Panel** and will allow the _Dispatcher_ to:

- Reorder _Tasks_ within a _Tour_ using _drag-n-drop_
- Add new _Tasks_ to a _Tour_ using _drag-n-drop_
- Create empty _Tours_
- Rename _Tours_
- Delete _Tours_

A _Tour_ can be assigned to a _Rider_ using _drag-n-drop_.

Some screenshots:

<details>
<summary>Image: vertical panels, all tours collapsed</summary>

![Screenshot 2023-05-11 at 11-25-11 CoopCycle](https://github.com/coopcycle/coopcycle-web/assets/667236/c2ac0d4d-f7bd-4775-9712-5c22703eb88a)
</details>

<details>
<summary>Image: vertical panels, tour expanded</summary>

![Screenshot 2023-05-11 at 11-25-47 CoopCycle](https://github.com/coopcycle/coopcycle-web/assets/667236/ae03b548-e3b2-41c7-aad6-994d073855f3)
</details>

<details>
<summary>Image: horizontal panels, some tours expanded</summary>

![Screenshot 2023-05-11 at 11-26-14 CoopCycle](https://github.com/coopcycle/coopcycle-web/assets/667236/16fc7287-e1b0-4fa1-9234-68031ea34f87)
</details>

<details>
<summary>Image: horizontal panels, some tours expanded, assigned tour</summary>

![Screenshot 2023-05-11 at 11-27-57 CoopCycle](https://github.com/coopcycle/coopcycle-web/assets/667236/316b7e5e-ef3f-4c2d-8cd5-cd1180b78b96)
</details>

**Comment by Paul-Eraman-CoopCycle** [comment]
@alexsegura This is completely true. However, to implement such a broad feature, there needs to be significantly more information here. Please elaborate about what this third dashboard panel would involve and how it many be represented.

**Comment by jimakker** [comment]
This already has a PR but I'm using that branch for https://github.com/coopcycle/coopcycle-web/issues/3597. I will discard the PR and we'll merge the other one.
